### DISCUSSION TITLE
How to extract UI5 verson at runtime?

### DISCUSSION THREAD

**Discussion by nair-sumesh** [ui5-community/wdi5#585] [Q&A] [answered]
Is there any wdi5 global function available to extract the underlying UI5 version being used by the app under test?

**Comment by vobu** [comment, marked as answer]
yessir, `browser.getUI5Version`:  https://github.com/ui5-community/wdi5/blob/d0d6e0f2b4f436adc43ea38f134a9466236fcc5b/src/types/browser-commands.ts#L27 !

**Reply by nair-sumesh** [reply]
Perfect. Thanks a lot.
